app/views/account/shared/_nav.html.slim

```diff
@@ -7,26 +7,19 @@
             i.ft-home
             span
               | Dashboard
-        - if current_user.admin?
+        - if current_user.admin? || current_user.super_admin?
```

Comment:
you are using this condition `admin? && super_admin?` in so many cases.
mb move it to a separate method on model? (e.g.: `current_user.god?`)